valohai/internals/download.py

```diff
@@ -1,8 +1,39 @@
 import os
 import tempfile
+from requests import Response
+from valohai.internals.utils import uri_to_filename, check_sha256_hashing
 
 
 # TODO: This is close to valohai-local-run. Possibility to merge.
+def resolve_datum(datum_id: str) -> Response:
+    try:
+        from valohai_cli.api import request  # type: ignore
+    except ImportError as ie:
+        raise RuntimeError("Can't resolve datum without valohai-cli") from ie
+    resp: Response = request(url=f"/api/v0/data/{datum_id}", method="GET", stream=True)
+    resp.raise_for_status()
+    return resp
+
+
+def verify_datum(response: Response, input_folder_path: str) -> str:
+    datum_obj = response.json()
+    filename = datum_obj["name"]
+    checksums = {
+        "md5": datum_obj["md5"],
+        "sha1": datum_obj["sha1"],
+        "sha256": datum_obj["sha256"],
+    }
```

Comment:
You're not using the other checksums than SHA256 anyway, so this isn't needed..?
```suggestion
```